_includes/recommendations.html

```diff
@@ -0,0 +1,17 @@
+<h2>Recommendations</h2>
+<div class="card-container">
+    {% for item in site.data.recommendations %}
+    <div class="card full">
+      <div class="icon">
+        <img src="./assets/img/recommendations/{{ item.logo }}">
+      </div>
+      <div class="info">
+        <p class="title">{{ item.title }}</p>
+        <p class="location">{{ item.author }}</p>
+        <p class="text">{{ item.text }}</p>
+        <button><a href="https://www.linkedin.com/in/{{ site.linkedin_username }}" target="_blank" rel="noopener">View more on LinkedIn</a></button>
+      </div>
```

Comment:
I think the `rel` should be `noopener noreferrer`
Can @robzwolf confirm

Reply:
Thanks, @jagnani73 for noticing that. Yes, it should be for wider browser support. Will amend.

Reply:
Have committed the changes. Thanks again.